srcpkgs/seahorse/patches/stdout.patch

```diff
@@ -0,0 +1,44 @@
+these keywords are reserved in C..?
```

Comment:
Use `https://gitlab.gnome.org/GNOME/seahorse/-/merge_requests/211.patch` instead, we are also interested in the header and author's description. Also I think this should be musl.patch because it's only an issue on musl afaict

Reply:
done